src/best_of/projects_collection.py

```diff
@@ -346,8 +346,12 @@ def apply_filters(project_info: Dict, configuration: Dict) -> None:
         project_info.show = False
 
     # Check platform requires
-    if configuration.require_github and not project_info.github_url:
-        log.info(f"{project_info.name} requires github, but no github repo found .")
+    if configuration.require_github and not (
```

Comment:
We should probably change that to `require_repo` to be a bit more generic.

Reply:
Yes, I also think that it would make sense. Do you think that this should all be done in this PR or should we merge soon and create a dedicated branch / PR?

Reply:
Yep, I think you can do this change in this PR as well.

Reply:
We should just keep in mind that this will break existing project.yaml files or should we add a fallback?

Reply:
ok, we can leave this in for one release. Once I have updated this in all our lists, we can remove this compatibility again (this will anyways not break anything).